examples/api/service-accounts.http

```diff
@@ -0,0 +1,21 @@
+### Login
+POST http://localhost:8080/api/v1/login
```

Comment:
This is a file we can create, and trigger from vscode , to test manually some of the endpoints.